charts/sn-platform/templates/vault/vault-statefulset.yaml

```diff
@@ -62,6 +62,7 @@ spec:
   {{- end }}
   {{- if or .Values.vault.config.bankVaults.probe.livenessProbe .Values.vault.config.bankVaults.probe.startupProbe .Values.vault.config.bankVaults.probe.readinessProbe }}
   vaultContainerSpec:
+    name: "{{ template "pulsar.fullname" . }}-{{ .Values.vault.component }}-vault-container-spec"
```

Comment:
it's the name of Vault container, shouldn't need the `container-spec` suffix.

Reply:
Fixed, thanks